package.json

```diff
@@ -732,6 +732,11 @@
         "title": "Text Editor",
         "order": 1,
         "properties": {
+          "dotnet.autoInsert.enableAutoInsert": {
```

Comment:
This needs to be covered by test.
https://github.com/dotnet/vscode-csharp/blob/main/test/lsptoolshost/unitTests/configurationMiddleware.test.ts